packages/protocol/contracts/governance/LockedGold.sol

```diff
@@ -323,28 +323,41 @@ contract LockedGold is ILockedGold, ReentrancyGuard, Initializable, UsingRegistr
     address[] calldata greaters,
     uint256[] calldata indices
   ) external onlySlasher {
-    uint256 maxSlash = Math.min(penalty, getAccountTotalLockedGold(account));
-    require(maxSlash >= reward, "reward cannot exceed penalty.");
+    uint256 actualPenalty = Math.min(
+      penalty,
+      getValidators().getAccountLockedGoldRequirement(account)
```

Comment:
I don't think we need to look at the requirement here, just the locked gold balance